OpenRA.Mods.Common/Projectiles/AreaBeam.cs

```diff
@@ -125,7 +125,7 @@ public AreaBeam(AreaBeamInfo info, ProjectileArgs args, Color color)
 			else
 				speed = info.Speed[0];
 
-			// Both the head and tail start at the source actor, but initially only the head is travelling.
+			// Both the head and tail start at the source actor, but initially only the head is traveling.
```

Comment:
Nit: two `l` is correct for British English and less common, but also not wrong for American English.
No blocker, but assuming this PR will need at least one more update anyway, imo just drop this while you're at it.

Reply:
Sorry, my spellchecker was complaining and I didn't think about that. Reverted.